testcert.py

```diff
@@ -3586,21 +3586,37 @@ def testJobStateWithPaperJam(self):
     test_id = '664a8841-14d0-483e-a91a-34722dfdb298'
     test_name = 'testJobStateWithPaperJam'
     print 'This test will validate job state when there is a paper jam.'
-    print 'Place page inside print path to cause a paper jam.'
-    raw_input('Select enter once printer reports paper jam.')
-    output = chrome.PrintFile(self.printer, Constants.IMAGES['PDF9'])
-    print 'Verify job is reported in error state.'
-    print 'Now clear the print path so the printer is no longer jammed.'
-    raw_input('Select enter once printer is clear of jam.')
-    print 'Verify print job prints after paper jam is cleared.'
-    try:
-      self.assertTrue(output)
-    except AssertionError:
-      notes = 'Error printing %s' % Constants.IMAGES['PDF9']
+    print 'Stick paper on the input tray which will cause a misfeed when feeding paper.'
+    raw_input('Select enter once it is sticked.')
+    if chrome.PrintFile(self.printer, Constants.IMAGES['PDF9']):
+      # give printer time to update our service.
+      time.sleep(10)
+      job_state = gcpmgr.WaitJobStatusNotIn('printtest.pdf', ['Queued', 'In progress'])
+      try:
+        self.assertEqual(job_state, 'Error')
+      except AssertionError:
+        notes = 'Print Job is not in Error state: %s' % job_state
+        self.LogTest(test_id, test_name, 'Failed', notes)
+        raise
+      else:
+        print 'Wait for the print job to finish.'
```

Comment:
Is the intention here to clear the paper jam and then let the print job finish? If so, we should place a comment in here that instructs the user to remove the paper jam.